packages/async-chunks/README.md

```diff
@@ -0,0 +1,71 @@
+# `@shopify/async-chunks`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fasync-chunks.svg)](https://badge.fury.io/js/%40shopify%2Fasync-chunks.svg)
+
+A code splitting & server side rendering solution for universal react apps inspired by [react-loadable](https://github.com/jamiebuilds/react-loadable).
+
+This package provides all the tools required to get your app code splitting & server side rendering in harmony:
+
+- Webpack plugin and Babel plugin
+
+- An HOC to generate asynchrously loadable chunks
+
+- Koa middleware
+
+- Initializers to preload chunks on the server & client
+
+## Installation
+
+```bash
+$ yarn add @shopify/async-chunks
+```
+
+## Usage
```

Comment:
Add step about webpack and babel plugins. Should we abstract this away and assume the consumers are using sewing-kit with the experiments flag enabled.